src/index.js

```diff
@@ -47,12 +47,52 @@ function rotateXYByAngle(pos, angle) {
 }
 
 const getTouchHandlerOption = props => {
-  if (props.touchHandlerOption) return props.touchHandlerOption
+  if (typeof props.touchHandlerOption !== 'undefined')
```

Comment:
also fixes this bug that should allow for `false` when options object with `passive` isn't supported.